src/app/main.cpp

```diff
@@ -30,6 +30,13 @@ int main(int argc, char ** argv) {
 
     // Create window.
     QApplication app(argc, argv);
+
+    // NOTE: set app variables which will be used to
+    // construct settings path
+    app.setOrganizationName("BaseALT");
+    app.setOrganizationDomain("basealt.ru");
+    app.setApplicationName("GPUI");
+    
```

Comment:
- [ ] Please move this change into new separate commit.
- [ ] Please provide conventional commit message to this commit: https://www.conventionalcommits.org/en/v1.0.0/ e.g. "feat: add organization name and domain to construct settings path"